core/src/main/java/com/shatteredpixel/shatteredpixeldungeon/items/weapon/missiles/MissileWeapon.java

```diff
@@ -44,9 +44,13 @@
 
 abstract public class MissileWeapon extends Weapon {
 
+	protected MissileWeapon( ) {
+		hitsToKnow = 2;
+	}
+
 	{
 		stackable = true;
-		levelKnown = true;
+		levelKnown = false;
```

Comment:
Items are currently only identified, if their level was unknown before. That makes it also a surprise to find secret +1 or even +2 missiles 0.7.1+ as a side effect.